pkg/didcomm/protocol/introduce/service.go

```diff
@@ -68,16 +69,20 @@ type Service struct {
 	service.Message
 	store     storage.Store
 	callbacks chan callback
+	sender    *providerImpl
 }
 
 // New returns introduce service
-func New(ctx context.Context, s storage.Provider) (*Service, error) {
+func New(ctx context.Context, s storage.Provider, d dispatcher.Outbound) (*Service, error) {
```

Comment:
We can use a provider similar to [DIDExchange Service](https://github.com/hyperledger/aries-framework-go/blob/master/pkg/didcomm/protocol/didexchange/service.go#L63).

Reply:
Done!